misc/docerina/src/main/java/org/ballerinalang/docgen/docs/BallerinaDocGenerator.java

```diff
@@ -666,7 +667,8 @@ private static void sortModuleConstructs(BLangPackage bLangPackage) {
         bLangPackage.getAnnotations().sort(Comparator.comparing(a -> a.getName().getValue()));
         bLangPackage.getTypeDefinitions()
                 .sort(Comparator.comparing(a -> a.getName() == null ? "" : a.getName().getValue()));
-        bLangPackage.getGlobalVariables().sort(Comparator.comparing(a -> a.getName().getValue()));
+        bLangPackage.getGlobalVariables().sort(Comparator.comparing(a ->
+                ((BLangSimpleVariable) a).getName().getValue()));
```

Comment:
Won't these casts leads to issues? Shouldn't we properly handle the list case also for now and appropriately handle the rest when they are being introduced?

Reply:
all the variables will desugar into simple variables, hence this should not fail